congress/base.py

```diff
@@ -28,12 +66,42 @@ def _response(self, resp):
         return self._parse(json_)
         
     def build_url(self, *args):
+        """Construct the proper URL from a list of arguments
+
+        Parameters
+        ----------
+        *args : Arguments
```

Comment:
maybe say something about how the `*args` are used in building the full query url